component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/backupcode/BackupCodeAuthenticator.java

```diff
@@ -567,37 +562,17 @@ private boolean isValidTokenLocalUser(String code, String username, Authenticati
     }
 
     private boolean isValidBackupCode(String token, AuthenticationContext context, String userName,
```

Comment:
The functionality of this method is wrong. The method says whether to check for validity. But we are removing the code the same time we validate the code.